versions/3.0.md

```diff
@@ -2365,42 +2370,147 @@ for the value that it is accompanying.  Tooling implementations MAY choose to
 validate compatibility automatically, and reject the example value(s) if they 
 are not compatible.
 
+Here is a single `example` in a model [Schema Object](#schemaObject):
+
+```json
+  "components" : {
+    "schemas" : {
+      "identity" : {
+        "properties" : {
+          "name" : {
+            "type" : "string"
+          }
+        },
+        "example" : {
+          "$ref" : "#/components/examples/identity"
+        }
+      }
+    }
+ }
+```
+
 ```yaml
-# in a model
-schemas:
-  properties:
-    name:
-      type: string
+components:
+  schemas:
+    identity:
+      properties:
+        name:
+          type: string
       example:
-        $ref: http://foo.bar#/examples/name-example
+        $ref: "#/components/examples/identity"
+```
+
+Below is an `examples` array in a [Request Body Object](#requestBodyObject),
```

Comment:
Same as above, you've already documented `examples` to be an _array_ so it seems unnecessary.